src/events/events.ts

```diff
@@ -310,6 +311,6 @@ export async function sendEvent(eventPayload: any) {
 
     // Don't halt program, but capture the failure in Sentry and Cloudwatch
     Sentry.captureException(failedEventError);
-    console.log(failedEventError);
+    serverLogger.error(failedEventError);
```

Comment:
similar here - we could send a parameterized error log. additionally, it would be good to differentiate this error from the one above (as they both currently have (i think) the exact same error message - which would make it difficult to know where the error actually happened).